storage/src/lib.rs

```diff
@@ -141,13 +141,15 @@ pub enum CheckerError {
 
     /// Hash mismatch for a node
     #[error(
-        "Hash mismatch for node {partial_path:?} at address {address}: parent stored {parent_stored_hash}, computed {computed_hash}"
+        "Hash mismatch for node {partial_path:?} at address {address}: parent ({parent:?}) stored {parent_stored_hash}, computed {computed_hash}"
```

Comment:
This looks pretty ugly I think. It's going to say "parent TrieNodeParent{Parent{LinearAddress{123}, 42}}}" or something like that.

I also think the partial path is not valuable at this point either.